components-ui/container/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactNode } from 'react';
+
+type ContainerProps = {
+  children: ReactNode;
+  className?: string;
+  style?: React.CSSProperties;
+};
+
+/**
+ * This container is used to wrap the main content of the page
+ *
+ * It can be wrapped inside another container to create a nested container, with a different style (usually, a container with a different background color)
+ *
+ */
+export default function Container({
+  children,
+  style = {},
+  className = '',
+}: ContainerProps) {
+  return (
```

Comment:
Even with the comment I am not sure I understand the purpose/need for this. 
We might want to use a slightly more obvious naming